graphql/server.go

```diff
@@ -157,9 +159,12 @@ func (c *conn) handleSubscribe(id string, subscribe *subscribeMessage) error {
 		ctx = c.makeCtx(ctx)
 
 		start := time.Now()
+		span, ctx := opentracing.StartSpanFromContext(ctx, "thunder.subscription")
+		ext.SamplingPriority.Set(span, 1)
```

Comment:
If it's a no-op, let's not include it.

Does the code run without problems if there is no global tracer setup?